src/NECompletion/NECModel.class.st

```diff
@@ -128,6 +128,16 @@ NECModel >> notEmpty [
 	^self isEmpty not
 ]
 
+{ #category : #printing }
+NECModel >> printOn: aStream [
+
+	super printOn: aStream.
+	clazz ifNotNil: 
+				[ aStream nextPutAll: ' ('.
+				clazz printOn: aStream.
+				aStream nextPutAll: ')' ]
```

Comment:
When pushing only one character on a Stream it is faster to use `aStream nextPut: $)`

It is only a detail here since I think it should not be call that often.